test/test.googleauth.ts

```diff
@@ -1390,21 +1390,21 @@ describe('googleauth', () => {
       const {auth, scopes} = mockGCE();
       mockEnvVar('GCLOUD_PROJECT', STUB_PROJECT);
       const email = 'google@auth.library';
-      const iamUri = `https://iam.googleapis.com`;
-      const iamPath = `/v1/projects/${STUB_PROJECT}/serviceAccounts/${email}:signBlob`;
-      const signature = 'erutangis';
+      const iamUri = `https://iamcredentials.googleapis.com`;
+      const iamPath = `/v1/%7Bname=projects/${STUB_PROJECT}/serviceAccounts/${email}%7D`;
```

Comment:
In Java we used the following pattern, (without  `{name=}`)

https://github.com/frankyn/google-auth-library-java/blob/master/oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java#L76

```java
  static final String SIGN_BLOB_URL_FORMAT = "https://iam.googleapis.com/v1/projects/-/serviceAccounts/%s:signBlob?alt=json";
```

Reply:
Happy to go change it. Just so we're doing this with eyes wide open, a few things:
- I chose this pattern based on the docs (here)[https://cloud.google.com/iam/docs/reference/credentials/rest/v1/projects.serviceAccounts/signBlob#http-request]. Is the doc correct?  
- I did write a sample, and made sure with an end to end test that the endpoint works.  So what's there now is effective (again happy to change)

Reply:
I followed the same document in Java and I think I mistook `{name=...}` as being an expression. Additionally, I find it easier to parse when reading it. 

For `STUB_PROJECT`, given the project id is in the service account, stating it explicitly seems unnecessary for me.